### ISSUE TITLE
[BUG]无法多客户端同时使用同一链接

### ISSUE THREAD

**Issue by Yiyoki** [sprov065/v2-ui#451]
出现状况是在面板创建vmess协议的链接后，该链接只能同时被至多一人使用，后连接上的设备无法使用。
创建过tcp和kcp协议均为默认配置创建。
vless等其他协议没有这个问题。

**Comment by yhussy** [comment]
Yes it well be great i you can fix that , also for Shadowsocks, Please)
